src/systems/filecoin_vm/actor/actor.go

```diff
@@ -11,7 +11,6 @@ type Serialization = util.Serialization
 const (
 	MethodSend        = MethodNum(0)
 	MethodConstructor = MethodNum(1)
-	MethodCron        = MethodNum(2)
```

Comment:
This is so we can explicitly include them as coming from the `cronactor`?

Reply:
This is debt. The CronActor has already been updated to be able to call any method registered with it. We don't need a special reserved method id.